Description: Remove unavailable dep network-interface, not used on Linux
Forwarded: not-needed
Last-Update: 2025-01-24
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -315,3 +315,3 @@
 ]
-network-interface = ["dep:network-interface"]
+network-interface = [] # ["dep:network-interface"]
 online-tests = []
@@ -324,3 +324,3 @@
 
-[target."cfg(not(any(target_os = \"android\", target_os = \"fuchsia\", target_os = \"linux\")))".dependencies.network-interface]
+[disabled.target."cfg(not(any(target_os = \"android\", target_os = \"fuchsia\", target_os = \"linux\")))".dependencies.network-interface]
 version = "1.0.0"
